core/arch/arm/include/smccc.h

```diff
@@ -0,0 +1,36 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2018, Linaro Limited
+ */
+
+#ifndef __SMCCC_H
+#define __SMCCC_H
+
+/*
+ * Describes features of SMC Calling Convention from v1.1
+ * See also https://developer.arm.com/-/media/developer/pdf/ARM%20DEN%200070A%20Firmware%20interfaces%20for%20mitigating%20CVE-2017-5715_V1.0.pdf
```

Comment:
ref is deprecated.

Looks like latest seems https://developer.arm.com/-/media/developer/pdf/ARM_DEN_0070A_Firmware_interfaces_for_mitigating_CVE-2017-5715.pdf?revision=d20d1476-b770-4739-aebe-2d3147788e3f&la=en.

But I think we should rather point to 
https://developer.arm.com/-/media/developer/pdf/ARM_DEN_0070A_Firmware_interfaces_for_mitigating_CVE-2017-5715.pdf